aws/checksum.go

```diff
@@ -0,0 +1,39 @@
+package aws
+
+// RequestChecksumCalculation controls request checksum calculation workflow
+type RequestChecksumCalculation string
+
+const (
+	// RequestChecksumCalculationWhenSupported indicates request checksum should be calculated if modeled
```

Comment:
"if modeled" does not mean anything to the caller. We have to find a different way to express this.